### PR TITLE
Atomic Transaction Support for Backend

### PR DESCRIPTION
**Pull request by poolcoke**

### CODE REVIEW

File: routes/server.go [hunk 1]
```diff
@@ -1419,6 +1422,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.LockedBalanceEntries,
 			PublicAccess,
 		},
+		{
+			"CreateAtomicTxnsWrapper",
+			[]string{"GET"},
```
Comment (lazynina) [inline routes/server.go:1427]: I think you want post and options.
```suggestion
			[]string{"POST", "OPTIONS"},
```

**Review by lazynina — CHANGES_REQUESTED** [review]

File: routes/server.go (hunk 1 above)
Comment (poolcoke) [reply routes/server.go:1427]: Yes you're definitely right. Will correct this.

**Review by tholonious — APPROVED** [review]

**Comment by lazynina** [comment]
* **#581** <a href="https://app.graphite.dev/github/pr/deso-protocol/backend/581?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#560** <a href="https://app.graphite.dev/github/pr/deso-protocol/backend/560?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @poolcoke and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by diamondhands0 — APPROVED** [review]

**Review by tholonious — APPROVED** [review]

**Review by lazynina — APPROVED** [review]

**Comment by lazynina** [comment]
### Merge activity

* **Apr 9, 2:46 PM EDT**: @lazynina started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/backend/560).

**Pull request merged**